### PR TITLE
Static pages

### PR DESCRIPTION
**Pull request by duccm94**

### CODE REVIEW

**Comment by qtsoul95** [comment]
Ok

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,14 +1,16 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title>MasterProject</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <head>
+  <title><%= yield :title %></title>
+  <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": true %>
```
Comment (daonhat) [inline app/views/layouts/application.html.erb]: ident

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
+  <%= javascript_include_tag "application", "data-turbolinks-track": true %>
```
Comment (daonhat) [inline app/views/layouts/application.html.erb]: ident

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -1,14 +1,16 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title>MasterProject</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <head>
+  <title><%= yield :title %></title>
```
Comment (daonhat) [inline app/views/layouts/application.html.erb]: ident

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (daonhat) [inline app/views/layouts/application.html.erb]: ident

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (daonhat) [inline app/views/layouts/application.html.erb]: ident

**Pull request merged**